config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl

```diff
@@ -485,9 +490,464 @@ ext:ProjectfaseringMetProjectverloopTimingEnRisicobeheerProjectGroup
 
 
 ##########################################################
-# TODO: Projectonderdelen
+# Projectonderdelen
 ##########################################################
 
+ext:projectOnderdeelPg a form:PropertyGroup;
+    sh:name "Projectonderdelen" ;
+    sh:order 10 .
```

Comment:
We need to revisit the order of the full form once everything is done, I think I used 10 as well since there was no "gap" for the TODO parts 😄.

Reply:
Yeah we will look at it later  :p